LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIStrategyMapItem_AlienNetworkComponent_LW.uc

```diff
@@ -0,0 +1,347 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    UIStrategyMapItem_AlienNetworkComponent_LW
+//  AUTHOR:  Joey Martinez -- 05/06/2019
+//  PURPOSE: This file represents an alien network spot on the StrategyMap for Long War of the Chosen.

```

Comment:
I would consider adding an explanation as to why we're overriding the base game class with this.

Reply:
done